### PR TITLE
[cli] rooch session-key list

### PR DESCRIPTION
**Pull request by vegetabledogdog**
```
rooch session-key list -h
List all session keys by address

Usage: rooch session-key list [OPTIONS]

Options:
  -a, --address <ADDRESS>        The account's address to list session keys, if absent, show the default active account [default: default]
      --password <PASSWORD>      The key store password
      --config-dir <CONFIG_DIR>  rooch config path
  -h, --help                     Print help
```

### CODE REVIEW

File: crates/testsuite/features/cmd.feature [hunk 1]
```diff
@@ -54,7 +54,8 @@ Feature: Rooch CLI integration tests
 
       # session key
       Then cmd: "session-key create  --app-name test --app-url https:://test.rooch.network --scope 0x3::empty::empty"
-      Then cmd: "move run --function 0x3::empty::empty  --session-key {{$.session-key[-1].authentication_key}}"
+      Then cmd: "session-key list"
```
Comment (jolestar) [inline crates/testsuite/features/cmd.feature:57]: Add an assert to ensure the command is executed successfully.

**Review by jolestar — APPROVED** [review]

File: crates/testsuite/features/cmd.feature (hunk 1 above)
Comment (vegetabledogdog) [reply crates/testsuite/features/cmd.feature:57]: The error message is like this.
```
rooch session-key list
Error: Session key not found
```
assert this
Then assert: "'{{$.session-key[-1]}}' not_contains Error"

File: frameworks/rooch-framework/sources/session_key.move [hunk 2]
```diff
@@ -260,6 +261,14 @@ module rooch_framework::session_key {
         remove_session_key(sender, authentication_key);
     }
 
+    public fun handle(account_address: address) : Option<ObjectID> {
```
Comment (jolestar) [inline frameworks/rooch-framework/sources/session_key.move:264]: Rename to `get_session_keys_handle`? More clear.

Comment (jolestar) [reply frameworks/rooch-framework/sources/session_key.move:264]: renamed in #2133

File: crates/rooch/src/commands/session_key/commands/list.rs [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        let address_addr = self.address.into_account_address(&mapping)?;
+
+        let function_id = "0x3::session_key::handle".parse::<FunctionId>()?;
+        let function_call = FunctionCall::new(function_id, vec![], vec![address_addr.to_vec()]);
+        let client = context.get_client().await?;
+        let view_result = client
+            .rooch
+            .execute_view_function(function_call)
+            .await
+            .map_err(|e| RoochError::ViewFunctionError(e.to_string()))?;
```
Comment (jolestar) [inline crates/rooch/src/commands/session_key/commands/list.rs:43]: This function call can be defined in the mod https://github.com/rooch-network/rooch/blob/main/crates/rooch-types/src/framework/session_key.rs

Then call it via client, like

https://github.com/rooch-network/rooch/blob/3197fd97aedc9d1ac76e974d0e029d1c47529f15/crates/rooch/src/commands/session_key/commands/create.rs#L102-L104

**Pull request merged**